clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java

```diff
@@ -629,28 +655,39 @@ private boolean handleVariablesResponse(@NonNull Response response) {
                     .processResponse(bodyJson, bodyString, context);
             return true;
         } else {
-            handleVariablesResponseError(response);
+            handleVarsOrTemplatesResponseError(response, "Variables");
             return false;
         }
     }
 
-    private void handleVariablesResponseError(Response response) {
+    private void handleVarsOrTemplatesResponseError(Response response, String logTag) {
```

Comment:
this is used only for logging, is there some wip workflow to add here?

Reply:
Nothing more is planned for now. In case of error, the backend will not apply any of the changes and we just log the error. This is the same behavior as with the /defineVars endpoint.